scout: add roles and permissions

dvdksn opened the pull request:
<!--
  Thank you for contributing to Docker documentation!

  Here are a few things to keep in mind:

  - Links between pages should use a relative path
  - Remember to add an alt text for images

  ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
  ┃     Review our contribution guidelines:       ┃
  ┃                                               ┃
  ┃ https://docs.docker.com/contribute/overview/  ┃
  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
-->

### Proposed changes

- chore: format with prettier
- scout: add roles and permissions

### Related issues (optional)

ENGDOCS-1627

stephaurelio reviewed (commented):
Nice ✨  Only thing I wonder is if the company owner needs to be part of the Scout table? We ended up removing that role from the first table as their permissions were more of a factor for org management.

dvdksn commented:
Thanks @stephaurelio, that makes sense - I've removed the company owner role from here. Also updated the permissions as per feedback from @neilprosser so this should be good to go

stephaurelio reviewed (approved):
LGTM!
